clamscan/manager.c

```diff
@@ -631,33 +631,325 @@ static int scanstdin(const struct cl_engine *engine, struct cl_scan_options *opt
     return ret;
 }
 
+struct sigload_progress {
+    time_t startTime;
+    time_t lastRunTime;
+    uint8_t bComplete;
+};
+
+struct engine_compile_progress {
+    time_t startTime;
+    time_t lastRunTime;
+    uint8_t bComplete;
+};
+
+struct engine_free_progress {
+    time_t startTime;
+    time_t lastRunTime;
+    uint8_t bComplete;
+};
+
+static void print_time(time_t seconds)
+{
+    if (seconds >= 3600) {
+        fprintf(stdout, "%2lldh %02lldm", (long long)seconds / 3600, ((long long)seconds % 3600) / 60);
+    } else if (seconds >= 60) {
+        fprintf(stdout, "%2lldm %02llds", (long long)seconds / 60, (long long)seconds % 60);
+    } else {
+        fprintf(stdout, "%3llds", (long long)seconds);
+    }
+}
+
+static void print_no_sigs(size_t sigs, int bPad)
```

Comment:
`print_num_sigs`? Nitpick, but it reads like `no`

Reply:
Yeah decent nitpick.  Will switch to `num`